model/trace/database.yaml

```diff
@@ -196,33 +224,37 @@ groups:
 
   - id: db.mongodb
     type: span
-    extends: attributes.db.client.minimal
+    extends: trace.db.common.minimal
     brief: >
       Attributes for MongoDB
     attributes:
       - ref: db.operation.name
+        sampling_relevant: true
         brief: >
           The name of the command being executed.
         note: >
           See [MongoDB database commands](https://www.mongodb.com/docs/manual/reference/command/).
         examples: ['findAndModify', 'getMore', 'update']
       - ref: db.collection.name
+        sampling_relevant: true
         brief:
           The MongoDB collection being accessed within the database stated in `db.namespace`.
         requirement_level: required
       - ref: db.namespace
+        sampling_relevant: true
         brief: The MongoDB database name.
         requirement_level:
           conditionally_required: If available.
         note: ""  # overriding the base note
 
   - id: db.elasticsearch
     type: span
-    extends: attributes.db.client.minimal
+    extends: trace.db.common.minimal
     brief: >
       Attributes for Elasticsearch
     attributes:
       - ref: http.request.method
+        sampling_relevant: true
         requirement_level: required
       - ref: db.operation.name
```

Comment:
This isn't marked with `sampling_relevant: true`, but it's listed as sampling relevant in the md file. We definitely have this during span creation.

I think `sampling_relevant: true` should be added here.

Reply:
> it's listed as sampling relevant in the md file

this means it's being inherited from an attribute group